iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -1108,12 +1110,12 @@ private void RemoveOldOperations(object _)
             _ = _twinResponseTimeouts
                 .Where(x => DateTimeOffset.UtcNow - x.Value > s_twinResponseTimeout)
                 .Select(x =>
-                    {
```

Comment:
I prefer this tabbing. The method body is a parameter, and as such should be indented under the call to `Select`.